modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -89,7 +97,7 @@ spec:
        syncOptions:
        - CreateNamespace=true
 ----
-<1> `path` is the branch that contains the `policygentemplates` CRs for the clusters.
+<1> `path` is the file path that contains the `policygentemplates` CRs for the clusters.
 <2> `repoURL` is the URL of the Git repository that contains the `policygentemplates` custom resources that specify configuration data for the site.
 <3> `targetRevision` is the branch on the Git repository that contains the relevant configuration data.
 
```

Comment:
While you are here, could you please add a `$` to `oc apply -k ./deployment`?

Reply:
Good catch Michael, $ added.